proposals/lip-0051.md

```diff
@@ -193,6 +193,7 @@ We further use the utility function `getMainchainID()` defined in [LIP 0037][lip
 | `TOKEN_ID_NOT_AVAILABLE` | uint32 | 11 | Used when the specified token ID is not available. |
 | `TOKEN_ID_NOT_NATIVE` | uint32 | 12 | Used when the specified token ID is not native to the chain. |
 | `INSUFFICIENT_ESCROW_BALANCE` | uint32 | 13 | Used when the escrowed account does not have sufficient balance. |
+| `INVALID_RECEIVING_CHAIN` | uint32 | 14 | Used when, during the cross-chain token transfer, the receiving chain is set to be equal to the sending chain. |
 
 ### Type Definition
```

Comment:
I leave my comment here but it concerns [supported tokens substore](https://github.com/LiskHQ/lips/blob/514f9fb029f3fb229d1ec7323417ad69ba08ade8/proposals/lip-0051.md#supported-tokens-substore).

I would modify the description of store keys like this:
"Each store key is the identifier of the chain to which the supported tokens are native: `chainID`, or `ALL_SUPPORTED_TOKENS_KEY`."

Reply:
Sounds good, I would do that (also for LIP 0052).

Reply:
Shouldn't this be resolved now?